### PR TITLE
Feature/airnow etl revisions

### PR DESCRIPTION
**Pull request by myshkins**
mostly revisions to airnow etl files so that they conform to waqi etl methods.

### CODE REVIEW

File: services/etl/dags/api_interface/get_airnow_data.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "format": "text/csv",
+    "verbose": "1",
+    "monitorType": "2",
+    "includerawconcentrations": "0",
+    "API_KEY": settings.AIRNOW_API_KEY,
+    }
+
+
+def get_airnow_data():
+    response = requests.get(AIRNOW_URL, params=params, timeout=20)
```
Comment (hucksy) [inline services/etl/dags/api_interface/get_airnow_data.py:27]: maybe we should put the api requests in try block?

File: services/etl/dags/etl_airnow_readings.py [hunk 2]
```diff
@@ -93,79 +95,61 @@ def reshape_airnow_data(df):
...
+    def drop_canada_rows():
+        """drops canada rows from readings"""
+        stmt = read_sql('dags/sql/drop_rows_airnow.sql')
+        with get_db() as db:
+            db.execute(stmt[0])
+            db.commit()
+
+    @task
+    def load_readings_prod_airnow():
+        """upserts airnow readings to prod table"""
```
Comment (hucksy) [inline services/etl/dags/etl_airnow_readings.py:138]: seems like we're repeating this execute sql block a bunch, also in my code, seems like we should move this to it's own function and just pass the sql script as a parameter.

File: services/etl/dags/etl_airnow_readings.py [hunk 3]
```diff
@@ -93,79 +95,61 @@ def reshape_airnow_data(df):
...
-        postgres_conn_id="postgres_etl_conn",
-        sql="sql/drop_rows_airnow.sql",
-    )
 
-    load_to_production = PostgresOperator(
-        task_id="airnow_load_to_prod_data",
-        postgres_conn_id="postgres_etl_conn",
-        sql="sql/load_prod_airnow_readings.sql",
-    )
+    a = create_table_temp_airnow()
```
Comment (hucksy) [inline services/etl/dags/etl_airnow_readings.py:145]: does this syntax enforce dependencies?

**Review by hucksy — APPROVED** [review]
Niice stuff dude, looks great to me. Maybe we can talk about putting the db execute block into it's own function, but otherwise good to merge.

File: services/etl/dags/etl_airnow_readings.py (hunk 3 above)
Comment (myshkins) [reply services/etl/dags/etl_airnow_readings.py:145]: right adding dependencies

**Pull request merged**